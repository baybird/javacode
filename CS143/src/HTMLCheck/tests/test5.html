<div>
    <h1></h1>
    <div>
        <img />
        <p><br/>
	  <br/>
	  <br/>        
    </div>
</div>
</table>
