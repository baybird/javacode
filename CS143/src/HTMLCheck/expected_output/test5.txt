<div>
    <h1></h1>
    <div>
        <img />
        <p><br />
	       <br />
	       <br />        
        </p>
    </div>
</div>
